dot/state/storage.go

```diff
@@ -114,13 +114,17 @@ func (s *StorageState) StoreTrie(root common.Hash, ts *rtstorage.TrieState) erro
 	ts.Close()
 
 	// store encoded *trie.Trie in database
-	err = s.StoreInDB(root)
-	if err != nil {
-		logger.Error("failed to store encoded trie in database", "state root", root, "error", err)
-		return err
-	}
+	// TODO: add to batch, write when sync is done or a certain amount of blocks are synced?
+	// in the future, should update how storage values are stored
+	go func() {
+		err = s.StoreInDB(root)
+		if err != nil {
+			logger.Error("failed to store encoded trie in database", "state root", root, "error", err)
+			return
+		}
 
-	logger.Trace("stored trie in database", "root", root)
+		logger.Trace("stored trie in database", "root", root)
+	}()
```

Comment:
Are we not depending on the possible error that `StoreInDB` could produce?

Reply:
we aren't really depending on it, this error generally shouldn't happen unless there's something wrong with the database, like it's been closed. if we want to handle the error we'd have to add some sort of err channel to get it out